### DISCUSSION TITLE
您好，我想问一下，训练完后，我想把自己的图片集放入进行测试该怎么修改代码，多谢！

### DISCUSSION THREAD

**Discussion by ChenJS95** [PengfeiM/Fatigue-Driven-Detection-Based-on-CNN#14] [Q&A]

**Comment by PengfeiM** [comment]
如果是测试单张图片的话修改这里的文件名
https://github.com/PengfeiM/Fatigue-Driven-Detection-Based-on-CNN/blob/cfa4f9a029e1ec40edf0b73d8618cc1c557f803f/Test.py#L24
输出得文件名在这里修改
https://github.com/PengfeiM/Fatigue-Driven-Detection-Based-on-CNN/blob/cfa4f9a029e1ec40edf0b73d8618cc1c557f803f/Test.py#L72

如果是用完整的数据集测试网络性能的话，用eval.py，请按照readme中的结构放置数据集图片的标签。
自己的图片集制作数据集请使用[LabelImg工具](https://github.com/tzutalin/labelImg)

**Comment by PengfeiM** [comment]
如果没啥问题，这个issue我明天就关了。

**Comment by ChenJS95** [comment]
感谢
